version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.4.0-SNAPSHOT"
+ThisBuild / projectVersion := "2.4.0-livongo-0.3.0"
```

Comment:
@jyerabutar  Will we able to publish a version like this in TGPS? Or are there any restrictions?

cc: @ctoomey